styles/_media.scss

```diff
@@ -0,0 +1,35 @@
+$mobile-width: 375px;
+$tablet-width: 768px;
+$tablet-xl-width: 1024px;
+$desktop-width: 1366px;
+$desktop-xl-width: 1920px;
```

Comment:
I think for now we can leave only a couple of breakpoints and add some of them later if we need them

Let's leave tablet-width and desktop-width